### PR TITLE
Add UI

### PR DESCRIPTION
**Pull request by HamiiidAsgarian**
I have added the foundation of the app UI. all the details would be fixed later after integrating other layers.

### CODE REVIEW

File: lib/view/screens/all_products_screen.dart [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+class AllProductsScreen extends StatelessWidget {
+  const AllProductsScreen({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+        appBar: const CustomAppbar(
+          title: "All Procusts",
+        ),
```
Comment (Codologyst) [inline lib/view/screens/all_products_screen.dart:16]: Fix the ratio after adding models.

File: lib/view/widgets/time_and_date.dart [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        Container(
+            padding: const EdgeInsets.symmetric(horizontal: 20),
+            height: 20,
+            child: const VerticalDivider(
+              thickness: 2,
+              width: 2,
+              color: AppConst.borderGrey,
+            )),
+        Text(date, style: AppConst.smallTextStyle)
+      ],
```
Comment (Codologyst) [inline lib/view/widgets/time_and_date.dart:42]: fix the date formatting after adding the data model.

File: lib/view/screens/profile_screen.dart [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+                              ),
+                            ),
+                          )))
+            ],
+          ),
+        ),
+      ),
+    );
+  }
+}
```
Comment (Codologyst) [inline lib/view/screens/profile_screen.dart:109]: Is this going to stay or new options will add up?

**Review by Codologyst — COMMENTED** [review]
It seems alright for this stage!

File: lib/view/screens/profile_screen.dart (hunk 3 above)
Comment (HamiiidAsgarian) [reply lib/view/screens/profile_screen.dart:109]: Some probably would be terminated :);

**Pull request merged**